src/test/fuzz/versionbits.cpp

```diff
@@ -203,7 +208,7 @@ FUZZ_TARGET_INIT(versionbits, initialize)
         }
 
         // don't risk exceeding max_blocks or times may wrap around
-        if (blocks.size() + period*2 > max_blocks) break;
+        if (blocks.size() + 2 * period > max_blocks) break;
     }
     // NOTE: fuzzed_data_provider may be fully consumed at this point and should not be used further
```

Comment:
https://github.com/bitcoin/bitcoin/pull/21380#discussion_r595114366 ;)

Reply:
The way it currently is makes way more sense to me; I had a go the other way to see if it'd make better use of fuzz magic somehow, but it doesn't; so sticking with what I like.